src/main/java/edu/harvard/iq/dataverse/MailServiceBean.java

```diff
@@ -111,7 +111,7 @@ public void sendMail(String host, String reply, String to, String subject, Strin
             me.printStackTrace(System.out);
         }
     }
-
+*/
     @Resource(name = "mail/notifyMailSession")
```

Comment:
Any reason we need to keep this fully commented out method around? (Also down around line 180)